Makefile

```diff
@@ -112,8 +112,8 @@ GOBIN=$(shell go env GOBIN)
 endif
 
 .PHONY: setup/moq
-setup/moq:
-	go install github.com/matryer/moq
+setup/moq: code/gen
+	GO111MODULE=off go get github.com/matryer/moq
```

Comment:
I think we shouldn't be changing `go install` to `go get` 
AFAIK:
`go install` installs moq binary to make it available via CLI
`go get` installs moq as library dependency

Reply:
Also, why is `code/gen` added above?

Reply:
I am sure Oleg, but on Monday this week or last week you came up with that fix we where having around moq. :confused:

Reply:
Yes, that's true, but later I realized my mistake and I thought we discussed it.
Sorry for the confusion.